main.tf

```diff
@@ -113,6 +113,26 @@ resource "aws_codebuild_project" "default" {
       "name"  = "GITHUB_TOKEN"
       "value" = "${var.github_token}"
     }
+
+    environment_variable {
+      "name"  = "JENKINS_URL"
```

Comment:
this is related to `Jenkins`, it should not be in this module which is supposed to be generic